docs/coordinates/stokescoord.rst

```diff
@@ -0,0 +1,86 @@
+.. _stokes-coord:
+
+Using the StokesCoord Class
+***************************
+
+The `.StokesCoord` class provides a minimal wrapper for interacting with Stokes parameters as handled by WCS.
+In WCS (as described in the FITS 4.0 specification), the stokes parameters are allocated an integer number, each representing a different parameter, for example "I" is given the number 1 and "Q" the number 2.
+The `.StokesCoord` class uses a mapping between these numbers and the names of the parameters as given by the FITS WCS conventions, and allows you to refer to the parameters by their string names rather than their numbers.
+For example, the default representation of the `.StokesCoord` object uses the names rather than the numbers.
+
+  >>> import numpy as np
+  >>> from astropy.coordinates import StokesCoord
+  >>> StokesCoord([1, 2, 3, 4])
+  StokesCoord(['I', 'Q', 'U', 'V'])
+
+These "symbols" as the `.StokesCoord` class refers to them as can also be accessed via the `.StokesCoord.symbol` property::
+
+  >>> stokes = StokesCoord([1, 2, 3, 4])
+  >>> stokes.symbol
+  array(['I', 'Q', 'U', 'V'], dtype='<U2')
+
+And the `.StokesCoord` class can also be instantiated with symbols rather than numbers:
+
+  >>> StokesCoord("I")
+  StokesCoord('I')
+
+Numeric values which are not present in the mapping will be represented by the string ``"?"``::
+
+  >>> StokesCoord([1, 10])
+  StokesCoord(['I', '?'])
+
+It is possible to add custom number - symbol mappings see :ref:`mapping-stokes-symbols`.
+
+
+Comparing to Parameters and Numbers
+-----------------------------------
+
+It is possible to compare the values in a `.StokesCoord` with their parameter names::
+
+  >>> stokes = StokesCoord([1, 2, 3, 4])
+  >>> stokes == "I"
+  array([ True, False, False, False])
+  >>> np.equal(stokes, "I")
```

Comment:
I think this is not needed in the documentation - direct use of `==` should be preferred. Also, `np.equal` is *much* slower since it converts every element to an object:
```
sc = StokesCoord(np.ones(1000))
%timeit sc == 1
# 1.68 µs ± 16.1 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
%timeit np.equal(sc, 1)
# 3.69 ms ± 21 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)
```